WorkspaceServer.Tests/NugetPackageKernelExtensionFinderTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using WorkspaceServer.Packaging;
+using Xunit;
+using System.IO;
+using MLS.Agent.Tools;
+using FluentAssertions;
+using System.Linq;
+using Microsoft.DotNet.Try.Markdown;
+
+namespace WorkspaceServer.Tests
+{
+    public class NugetPackageKernelExtensionFinderTests
+    {
+        [Fact]
+        public void Gets_all_extensions_dlls_inside_nuget_package_folder()
+        {
+            var nugetpackageName = "myNuget";
+            var directory = new DirectoryInfo(Directory.GetCurrentDirectory()).Subdirectory(nugetpackageName);
```

Comment:
Any reason to use `Directory.GetCurrentDirectory` rather than hard-coding a dummy path?